## Análise dos antecedentes dos congressistas brasileiros pelo olhar ambiental

### A base das análises é a planilha 'políticos_check_trabalho.csv' que foi montada a partir do cruzamento de várias fontes de dados como http://www.politicos.org.br/processos; STF; tribunais estaduais e eleitorais para avaliar os congressistas sobre processos cíveis, criminais, eleitorais em que são réus e condenados.

#### A primeira parte da análise buscou processos por dano ambiental.

In [1]:
import pandas as pd

In [2]:
ambiente = pd.read_csv('politicos_check_trabalho.csv')

In [3]:
ambiente.head()

Nome             cargo partido estado  \
0           Acir Gurgacz           Senador     PDT     RO   
1  Adalberto Cavalcanti   Deputado Federal  AVANTE     PE   
2  Adalberto Cavalcanti   Deputado Federal  AVANTE     PE   
3  Adalberto Cavalcanti   Deputado Federal  AVANTE     PE   
4  Adalberto Cavalcanti   Deputado Federal  AVANTE     PE   

                   ocorrencia  \
0                 Estelionato   
1  Improbidade Administrativa   
2  Improbidade Administrativa   
3              Dano ao Erário   
4  Improbidade Administrativa   

                                           descricao                  status  \
0  Crime de Estelionato e crimes contra o Sistema...   Em andamento- Recurso   
1  Emprego irregular de verbas ou rendas públicas...  Em andamento - Recurso   
2                         Improbidade Administrativa            Em Andamento   
3                         Improbidade Administrativa          Dano ao Erário   
4                         Improbidade Administrativa            Em Andamento   

                                            processo  
0  935 - STF (http://politica.estadao.com.br/noti...  
1  976/2016 - STF (http://portal.stf.jus.br/proce...  
2  0000323-39.2010.8.17.0120 - TJ - PE/ Comarca d...  
3  0000015-32.2012.8.17.0120 - TJ - PE/ Comarca d...  
4  0003473-41.2016.8.17.2990 - TJ - PE/ Comarca d...

In [4]:
ambiente.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1294 entries, 0 to 1293
Data columns (total 8 columns):
Nome          1294 non-null object
cargo         1294 non-null object
partido       1294 non-null object
estado        1294 non-null object
ocorrencia    1294 non-null object
descricao     1294 non-null object
status        1261 non-null object
processo      1261 non-null object
dtypes: object(8)
memory usage: 81.0+ KB


In [5]:
ambiente[ambiente.Nome == 'Macedo']

Nome             cargo partido estado      ocorrencia  \
628  Macedo  Deputado Federal      PP     CE  Dano Ambiental   
629  Macedo  Deputado Federal      PP     CE  Dano Ambiental   
630  Macedo  Deputado Federal      PP     CE  Dano Ambiental   

                                             descricao              status  \
628  Foi autuado pelo Ministério Público Federal po...  Segredo de Justiça   
629    É alvo de ação civil pública por Dano Ambiental  Segredo de Justiça   
630  Acusado de construir imóvel residencial  em ár...               Findo   

                                              processo  
628  0000266-74.2008.4.05.8101 - TRF - 5 (http://ww...  
629  0006293-66.2014.4.05.8100 - TRF - 5 (http://ww...  
630  980 - STF (http://portal.stf.jus.br/processos/...

#### 20 processos por danos ambientais foram encontrados, a maioria por transformar áreas públicas, de preservação ambiental e áreas em particulares, construindo edificações irregulares e degradando o ambiente ao redor. A maior parte dos processos termina em acordo com pagamento de multas ambientais e/ou a obrigação de reparar o dano, o que quase nunca acontece.


In [6]:
ambiente[ambiente.ocorrencia == 'Dano Ambiental']

Nome             cargo partido estado      ocorrencia  \
5     Adelmo Carneiro Leão  Deputado Federal      PT     MG  Dano Ambiental   
171       Augusto Coutinho  Deputado Federal      SD     PE  Dano Ambiental   
172       Augusto Coutinho  Deputado Federal      SD     PE  Dano Ambiental   
173       Augusto Coutinho  Deputado Federal      SD     PE  Dano Ambiental   
209         Carlos Bezerra  Deputado Federal     MDB     MT  Dano Ambiental   
281         Dâmina Pereira  Deputado Federal    PODE     MG  Dano Ambiental   
326           Edmar Arruda  Deputado Federal     PSD     PR  Dano Ambiental   
483           Irajá Abreu   Deputado Federal     PSD     TO  Dano Ambiental   
507        Jandira Feghali  Deputado Federal   PCdoB     RJ  Dano Ambiental   
563        João Paulo Papa  Deputado Federal    PSDB     SP  Dano Ambiental   
628                 Macedo  Deputado Federal      PP     CE  Dano Ambiental   
629                 Macedo  Deputado Federal      PP     CE  Dano Ambiental   
630                 Macedo  Deputado Federal      PP     CE  Dano Ambiental   
669        Renan Calheiros           Senador     MDB     AL  Dano Ambiental   
712            Kátia Abreu           Senador     PDT     TO  Dano Ambiental   
810         Paulo Teixeira  Deputado Federal      PT     SP  Dano Ambiental   
866          Marcio Alvino  Deputado Federal      PR     SP  Dano Ambiental   
1002    Nelson Marquezelli  Deputado Federal     PTB     SP  Dano Ambiental   
1011    Newton Cardoso Jr   Deputado Federal     MDB     MG  Dano Ambiental   
1226               Rossoni  Deputado Federal    PSDB     PR  Dano Ambiental   

                                              descricao  \
5                                        Dano Ambiental   
171   Crimes contra o Meio Ambiente e o Patrimônio G...   
172   É alvo de ação civil pública movida pelo Minis...   
173   É réu em ação penal sobre crimes contra o meio...   
209   Foi condenado por Dano Ambiental. O juiz deter...   
281   Crimes contra o Meio Ambiente e o Patrimônio G...   
326                                      Dano Ambiental   
483   Foi multado pelo IBAMA por desmatar vegetação ...   
507   Dano Ambiental, indenização por Dano Moral, re...   
563                  Unidade de conservação da natureza   
628   Foi autuado pelo Ministério Público Federal po...   
629     É alvo de ação civil pública por Dano Ambiental   
630   Acusado de construir imóvel residencial  em ár...   
669   Foi condenado a apresentar plano de compensaçã...   
712               Revogação/Anulação de multa ambiental   
810                   Ação relacionada ao meio ambiente   
866   É réu em ação civil pública referente a crimes...   
1002  É réu em ação civil pública por Dano Ambiental...   
1011  É réu em ação penal por crimes contra o meio a...   
1226  O parlamentar é investigado por crimes contra ...   

                                                 status  \
5     Processo suspenso por convenção das partes par...   
171                                        Em Andamento   
172                                        Em Andamento   
173                                        Em Andamento   
209                                        Em Andamento   
281                                           Inquérito   
326            Ação aceita, aguarda decisão em 1o grau.   
483                                        Em andamento   
507                                        Em andamento   
563                                        Em andamento   
628                                  Segredo de Justiça   
629                                  Segredo de Justiça   
630                                               Findo   
669                                        Em andamento   
712                Findo- perpetua-se a multa ambiental   
810                                        Em andamento   
866                                            Suspenso   
1002                             Em andamento / Recurso   
1

#### Atualização dos deputados que cometeram crime ambiental.
 

In [7]:
x = ['Washigton Reis', 'Deputado Federal', 'MDB', 'RJ', 'Dano Ambiental', 
     'Réu por crimes contra o meio ambiente pela construção, instalação e funcionamento de estabelecimentos poluidores, capazes de causar degradação ambiental, sem prévio licenciamento ambiental',
     'Em Andamento', 'STF - http://portal.stf.jus.br/processos/detalhe.asp?incidente=4094146']  
#fonte Ruralômetro

In [8]:
x

['Washigton Reis',
 'Deputado Federal',
 'MDB',
 'RJ',
 'Dano Ambiental',
 'Réu por crimes contra o meio ambiente pela construção, instalação e funcionamento de estabelecimentos poluidores, capazes de causar degradação ambiental, sem prévio licenciamento ambiental',
 'Em Andamento',
 'STF - http://portal.stf.jus.br/processos/detalhe.asp?incidente=4094146']

In [9]:
crime_ambiental = ambiente[ambiente.ocorrencia == 'Dano Ambiental']

In [10]:
crime_ambiental['estado'].value_counts()

SP    4
PE    3
CE    3
MG    3
PR    2
TO    2
RJ    1
MT    1
AL    1
Name: estado, dtype: int64

In [11]:
crime_ambiental['partido'].value_counts()

SD       3
MDB      3
PP       3
PSD      2
PSDB     2
PT       2
PDT      1
PR       1
PCdoB    1
PTB      1
PODE     1
Name: partido, dtype: int64

In [12]:
crime_ambiental.groupby('partido')['Nome'].value_counts()

partido  Nome                
MDB      Carlos Bezerra          1
         Newton Cardoso Jr       1
         Renan Calheiros         1
PCdoB    Jandira Feghali         1
PDT      Kátia Abreu             1
PODE     Dâmina Pereira          1
PP       Macedo                  3
PR       Marcio Alvino           1
PSD      Edmar Arruda            1
         Irajá Abreu             1
PSDB     João Paulo Papa         1
         Rossoni                 1
PT       Adelmo Carneiro Leão    1
         Paulo Teixeira          1
PTB      Nelson Marquezelli      1
SD       Augusto Coutinho        3
Name: Nome, dtype: int64

In [13]:
crime_ambiental.groupby('Nome')['ocorrencia'].value_counts()

Nome                  ocorrencia    
Adelmo Carneiro Leão  Dano Ambiental    1
Augusto Coutinho      Dano Ambiental    3
Carlos Bezerra        Dano Ambiental    1
Dâmina Pereira        Dano Ambiental    1
Edmar Arruda          Dano Ambiental    1
Irajá Abreu           Dano Ambiental    1
Jandira Feghali       Dano Ambiental    1
João Paulo Papa       Dano Ambiental    1
Kátia Abreu           Dano Ambiental    1
Macedo                Dano Ambiental    3
Marcio Alvino         Dano Ambiental    1
Nelson Marquezelli    Dano Ambiental    1
Newton Cardoso Jr     Dano Ambiental    1
Paulo Teixeira        Dano Ambiental    1
Renan Calheiros       Dano Ambiental    1
Rossoni               Dano Ambiental    1
Name: ocorrencia, dtype: int64

#### Com informações do relatório da Amazon Watch acusando seis deputados da bancada ruralista de crimes ambientais, sociais e econômicos relacionados à Amazônia brasileira. Cinco deles são candidatos às eleições 2018. São apontados Adilton Sachetti (PRB/MT), Alfredo Kaefer (PP/PR), Dilceu Sperafico (PP/PR), Nelson Marquezelli (PTB/SP),Adilton Sachetti (PRB/MT) e o deputados estadual Jorge Amanajás (PPS/AP) que não faz parte dessa análise.

#### Fonte: https://jornalggn.com.br/sites/default/files/documentos/2018-complicity-in-destruction.pdf

In [14]:
ambiente[ambiente.Nome == 'Adilton Sachetti']

Nome             cargo partido estado  \
10  Adilton Sachetti  Deputado Federal     PRB     MT   
11  Adilton Sachetti  Deputado Federal     PRB     MT   
12  Adilton Sachetti  Deputado Federal     PRB     MT   
13  Adilton Sachetti  Deputado Federal     PRB     MT   
14  Adilton Sachetti  Deputado Federal     PRB     MT   
15  Adilton Sachetti  Deputado Federal     PRB     MT   
16  Adilton Sachetti  Deputado Federal     PRB     MT   
17  Adilton Sachetti  Deputado Federal     PRB     MT   
18  Adilton Sachetti  Deputado Federal     PRB     MT   
19  Adilton Sachetti  Deputado Federal     PRB     MT   
20  Adilton Sachetti  Deputado Federal     PRB     MT   
21  Adilton Sachetti  Deputado Federal     PRB     MT   
22  Adilton Sachetti  Deputado Federal     PRB     MT   
23  Adilton Sachetti  Deputado Federal     PRB     MT   

                          ocorrencia  \
10        Improbidade Administrativa   
11                   Crime Eleitoral   
12         Crime de Responsabilidade   
13         Crime de Responsabilidade   
14         Crime de Responsabilidade   
15        Improbidade Administrativa   
16  Crime contra a Lei de Licitações   
17              Apropriação Indébita   
18              Apropriação indébita   
19                    Dano ao Erário   
20                    Dano ao Erário   
21              Apropriação Indébita   
22  Crime contra a Lei de Licitações   
23            Enriquecimento Ilícito   

                                            descricao  \
10  O parlamentar é acusado de realiza repasses ir...   
11         Falsidade Ideológica para fins eleitorais.   
12                          Crime de Responsabilidade   
13  Crime de Responsabilidade enquadrado na Lei de...   
14                          Crime de Responsabilidade   
15  O deputado é de Improbidade Administrativa pel...   
16  O Tribunal constatou irregularidade em contrat...   
17  A acusação alega que o ex-prefeito se omitiu a...   
18  O Deputado é acusado de desviar verba recebida...   
19  Omissão do deputado, à época Prefeito, na insp...   
20  O parlamentar é acusado de causar dano aos cof...   
21  O ex-prefeito é acusado de omissão ao permitir...   
22  A acusação alega que foram transferidos dois l...   
23  A acusação alega que foram transferidos lotes ...   

                            status  \
10                    Em Andamento   
11                    Em Andamento   
12  Em Andamento - Denúncia Aceita   
13                    Em Andamento   
14                    Em Andamento   
15                      Em Recurso   
16                      Prescrição   
17                    Em Andamento   
18                    Em Andamento   
19                    Em Andamento   
20                    Em Andamento   
21                    Em Andamento   
22                    Em Andamento   
23                    Em Andamento   

                                             processo  
10  Ação Penal 947/2015 - STF (http://portal.stf.j...  
11  4165/2015 - STF (http://portal.stf.jus.br/proc...  
12  4210/2015 - STF (http://portal.stf.jus.br/proc...  
13  4632/2017 - STF (http://portal.stf.jus.br/proc...  
14  4310/2016 - STF (http://portal.stf.jus.br/proc...  
15  0004093-05.2009.8.11.0003 - TJ-MT (http://serv...  
16  nº 103870/2008 - TCE-MT (http://www.tce.mt.gov...  
17  8931-83.2012.811.0003 - TJ-MT - Comarca de Ron...  
18  13082-92.2012.811.0003 - TJ-MT - Comarca de Ro...  
19  14048-21.2013.811.0003 - TJ-MT - Comarca de Ro...  
20  2655-94.2016.811.0003 - TJ-MT - Comarca de Ron...  
21  8931-83.2012.811.0003 - TJ-MT - Comarca de Ron...  
22  14705-60.2013.811.0003 - TJ-MT - Comarca de Ro...  
23  15733-63.2013.811.0003 - TJ-MT - Comarca de Ro...

In [15]:
adilton = ambiente[ambiente.Nome == 'Adilton Sachetti']

In [16]:
adilton.ocorrencia.value_counts()

Crime de Responsabilidade           3
Dano ao Erário                      2
Crime contra a Lei de Licitações    2
Improbidade Administrativa          2
Apropriação Indébita                2
Crime Eleitoral                     1
Enriquecimento Ilícito              1
Apropriação indébita                1
Name: ocorrencia, dtype: int64

In [17]:
adilton.Nome.count()

14

In [18]:
#O ex-prefeito é acusado de omissão ao permitir que seu secretário municipal de Desenvolvimento Econômico se apropriasse irregularmente de lotes públicos no Distrito Industrial de Rondonópolis por meio de empresa fictícia, o que causou prejuízo aos cofres públicos.
#A acusação alega que foram transferidos dois lotes públicos à Construtora Égide sem realização de licitação, causando prejuízo aos cofres públicos.
#A acusação alega que foram transferidos lotes públicos às empresas Sachet & Fagundes Ltda. e Agropecuária B&Q S.A. sem procedimento licitatório, enriquecendo-as ilicitamente e causando prejuízo aos cofres públicos.


In [19]:
ambiente[ambiente.Nome == 'Nelson Marquezelli']   #Junto com Companhia Siderúrgica Pitangui e Rio Rancho Agropecuária S/A, 
                                                  #de propriedade de sua família, o deputado é réu por falsidade ideológica e por crimes contra o meio ambiente.


Nome             cargo partido estado  \
1000  Nelson Marquezelli  Deputado Federal     PTB     SP   
1001  Nelson Marquezelli  Deputado Federal     PTB     SP   
1002  Nelson Marquezelli  Deputado Federal     PTB     SP   
1003  Nelson Marquezelli  Deputado Federal     PTB     SP   

                      ocorrencia  \
1000             Crime Eleitoral   
1001             Crime Eleitoral   
1002              Dano Ambiental   
1003  Improbidade Administrativa   

                                              descricao  \
1000  Contas de candidato a deputado federal desapro...   
1001  Contas de campanha eleitoral para deputado fed...   
1002  É réu em ação civil pública por Dano Ambiental...   
1003  É réu em ação civil pública por Improbidade Ad...   

                      status  \
1000                 Julgado   
1001                 Julgado   
1002  Em andamento / Recurso   
1003  Em andamento / Recurso   

                                               processo  
1000  0005709-96.2014.6.26.0000 - TRE - SP (http://i...  
1001  0010530-85.2010.6.26.0000 - TRE - SP (http://i...  
1002  0007484-05.2013.8.26.0457 (045.72.0130.007484)...  
1003  1000767-42.2015.8.26.0601 - TJ - SP (https://e...

In [20]:
ambiente[ambiente.Nome == 'Gabriel Guimarães']      # quem mais recebeu doação eleitoral direta e indireta de empresas flagradas por trabalho escravo.


Empty DataFrame
Columns: [Nome, cargo, partido, estado, ocorrencia, descricao, status, processo]
Index: []

In [21]:
ambiente[ambiente.Nome == 'Alfredo Kaefer']

Nome             cargo partido estado  \
490  Alfredo Kaefer  Deputado Federal      PP     PR   
491  Alfredo Kaefer  Deputado Federal      PP     PR   
492  Alfredo Kaefer  Deputado Federal      PP     PR   
493  Alfredo Kaefer  Deputado Federal      PP     PR   
494  Alfredo Kaefer  Deputado Federal      PP     PR   
495  Alfredo Kaefer  Deputado Federal      PP     PR   

                            ocorrencia  \
490                Crimes Falimentares   
491  Crimes contra a Ordem Tributária.   
492                Lavagem de Dinheiro   
493  Crimes contra a Ordem Tributária.   
494  Crime contra o Patrimônio Público   
495              Formação de Quadrilha   

                                             descricao        status  \
490                                Crimes Falimentares  Em andamento   
491                  Crimes contra a Ordem Tributária.  Em andamento   
492  Crimes de "Lavagem" ou Ocultação de Bens, Dire...  Em andamento   
493                  Crimes contra a Ordem Tributária.  Em andamento   
494                  Crime contra o Patrimônio Público  Em andamento   
495  É réu em ação penal por crimes contra o sistem...  Em andamento   

                                              processo  
490  4298/2016 - STF (http://www.stf.jus.br/portal/...  
491  4304/2016 - STF (http://www.stf.jus.br/portal/...  
492  4675/2018 - STF (http://portal.stf.jus.br/proc...  
493  3809/2012 - STF (http://www.stf.jus.br/portal/...  
494  3678/2013 - STF (http://www.stf.jus.br/portal/...  
495  892/2014 - STF (http://www.stf.jus.br/portal/p...

In [22]:
alfredo = ambiente[ambiente.Nome == 'Alfredo Kaefer']

In [23]:
alfredo.Nome.count()

6

In [24]:
ambiente[ambiente.Nome == 'Sidney Rosa']

Empty DataFrame
Columns: [Nome, cargo, partido, estado, ocorrencia, descricao, status, processo]
Index: []

#### Cruzamento e análise de dados da planilha bancadas da camara disponibilizada em matéria da Agencia Publica 
#### Fonte: https://apublica.org/2016/02/truco-as-bancadas-da-camara/

In [25]:
agro = pd.read_csv('bancadas_camara.csv')

In [26]:
agro.head()

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
0   Alexandre Leite     DEM     SP       2.0  Reeleito         NaN        1.0   
1     Carlos Melles     DEM     MG       6.0  Reeleito         1.0        1.0   
2    Cláudio Cajado     DEM     BA       6.0  Reeleito         NaN        1.0   
3      Efraim Filho     DEM     PB       3.0  Reeleito         NaN        1.0   
4  Eli Corrêa Filho     DEM     SP       2.0  Reeleito         NaN        1.0   

   Bala  Sindical  Direitos Humanos  Mineração  Empresarial  Parentes  Saúde  \
0   NaN       NaN               NaN        NaN          1.0       1.0    NaN   
1   NaN       NaN               NaN        NaN          1.0       1.0    NaN   
2   NaN       NaN               NaN        1.0          1.0       1.0    NaN   
3   1.0       NaN               NaN        NaN          NaN       1.0    NaN   
4   NaN       NaN               NaN        NaN          1.0       NaN    NaN   

   Bola  Empreiteiras/construtoras  Total de bancadas  
0   NaN                        1.0                4.0  
1   NaN                        1.0                5.0  
2   NaN                        1.0                5.0  
3   NaN                        1.0                4.0  
4   NaN                        1.0                3.0

In [27]:
agro.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 513 entries, 0 to 512
Data columns (total 17 columns):
Nome                         513 non-null object
partido                      512 non-null object
estado                       512 non-null object
mandatos                     512 non-null float64
Situação                     335 non-null object
Evangélica                   196 non-null float64
Ruralista                    207 non-null float64
Bala                         35 non-null float64
Sindical                     44 non-null float64
Direitos Humanos             25 non-null float64
Mineração                    24 non-null float64
Empresarial                  209 non-null float64
Parentes                     239 non-null float64
Saúde                        22 non-null float64
Bola                         15 non-null float64
Empreiteiras/construtoras    226 non-null float64
Total de bancadas            512 non-null float64
dtypes: float64(13), object(4)
memory usage: 68.2+ KB


In [28]:
agro['Ruralista'].count()     

207

In [29]:
agro[(agro['Ruralista'] == 1)]

Nome partido estado  mandatos  Situação  Evangélica  \
0           Alexandre Leite     DEM     SP       2.0  Reeleito         NaN   
1             Carlos Melles     DEM     MG       6.0  Reeleito         1.0   
2            Cláudio Cajado     DEM     BA       6.0  Reeleito         NaN   
3              Efraim Filho     DEM     PB       3.0  Reeleito         NaN   
4          Eli Corrêa Filho     DEM     SP       2.0  Reeleito         NaN   
7               Hélio Leite     DEM     PA       1.0       NaN         1.0   
10                 Mandetta     DEM     MS       2.0  Reeleito         NaN   
13           Misael Varella     DEM     MG       1.0      Novo         NaN   
15           Ônyx Lorenzoni     DEM     RS       4.0  Reeleito         NaN   
30             Afonso Motta     PDT     RS       2.0       NaN         NaN   
31                Dagoberto     PDT     MS       2.0       NaN         NaN   
33    Félix Mendonça Júnior     PDT     BA       2.0  Reeleito         NaN   
34            Flávia Morais     PDT     GO       2.0       NaN         NaN   
36          Giovani Cherini     PDT     RS       2.0       NaN         NaN   
40           Mário Heringer     PDT     MG       4.0       NaN         1.0   
42             Roberto Góes     PDT     AP       1.0       NaN         NaN   
44           Sergio Vidigal     PDT     ES       1.0      Novo         1.0   
46           Weverton Rocha     PDT     MA       2.0       NaN         NaN   
48             André Fufuca     PEN     MA       1.0       NaN         NaN   
51             Diego Garcia     PHS     PR       1.0       NaN         1.0   
52            Kaio Maniçoba     PHS     PE       1.0       NaN         1.0   
53              Marcelo Aro     PHS     MG       1.0       NaN         NaN   
54     Abel Mesquita Júnior     PMB     RR       1.0      Novo         NaN   
58           Assis do Couto     PMB     PR       4.0  Reeleito         NaN   
60   Carlos Henrique Gaguim     PMB     TO       1.0      Novo         1.0   
62            Domingos Neto     PMB     CE       2.0  Reeleito         1.0   
63            Fábio Ramalho     PMB     MG       3.0  Reeleito         NaN   
65          Juscelino Filho     PMB     MA       1.0       NaN         NaN   
68   Marcelo Álvaro Antônio     PMB     MG       1.0      Novo         1.0   
72         Valtenir Pereira     PMB     MT       3.0  Reeleito         1.0   
..                      ...     ...    ...       ...       ...         ...   
378                   Rocha    PSDB     AC       1.0       NaN         1.0   
410       Gabriel Guimarães      PT     MG       2.0  Reeleito         NaN   
421       Leonardo Monteiro      PT     MG       4.0  Reeleito         NaN   
438             Pedro Uczai      PT     SC       2.0  Reeleito         1.0   
443           Ságuas Moraes      PT     MT       2.0       NaN         NaN   
444            Sibá Machado      PT     AC       2.0  Reeleito         NaN   
448         Vicente Cândido      PT     SP       2.0       NaN         NaN   
457           Alex Canziani     PTB     PR       5.0  Reeleito         NaN   
461             Benito Gama     PTB     BA       5.0       NaN         NaN   
466          Josué Bengtson     PTB     PA       4.0  Reeleito         1.0   
467          Jovair Arantes     PTB     GO       6.0  Reeleito         NaN   
468             Jozi Araújo     PTB     AP       1.0      Novo         NaN   
470      Nelson Marquezelli     PTB     SP       7.0  Reeleito         1.0   
471         Nilton Capixaba     PTB     RO       4.0  Reeleito         1.0   
472             Paes Landim     PTB     PI       8.0       NaN         NaN   
476            Wilson Filho     PTB     PB       2.0  Reeleito         NaN   
486           Evair de Melo      PV     ES       1.0       NaN         NaN   
487           Evandro Gussi      PV     SP       1.0       NaN         1.0   
496    Arthur Oliveira Maia      SD     BA       2.0  Reeleito         NaN   
497        Augusto Carvalho      SD     DF   

In [30]:
rural = agro[(agro['Ruralista'] == 1)]

In [31]:
rural.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 206 entries, 0 to 510
Data columns (total 17 columns):
Nome                         206 non-null object
partido                      206 non-null object
estado                       206 non-null object
mandatos                     206 non-null float64
Situação                     149 non-null object
Evangélica                   83 non-null float64
Ruralista                    206 non-null float64
Bala                         17 non-null float64
Sindical                     7 non-null float64
Direitos Humanos             0 non-null float64
Mineração                    18 non-null float64
Empresarial                  113 non-null float64
Parentes                     105 non-null float64
Saúde                        5 non-null float64
Bola                         7 non-null float64
Empreiteiras/construtoras    91 non-null float64
Total de bancadas            206 non-null float64
dtypes: float64(13), object(4)
memory usage: 29.0+ KB


#### Segundo esses dados 'http://bit.ly/2Q9Rr4h' a bancada ruralista na Câmara dos Deputados é composta de 206 integrantes.

In [32]:
rural.Situação.value_counts()

Reeleito    113
Novo         31
Suplente      5
Name: Situação, dtype: int64

In [33]:
rural.Nome

0             Alexandre Leite
1               Carlos Melles
2              Cláudio Cajado
3                Efraim Filho
4            Eli Corrêa Filho
7                 Hélio Leite
10                   Mandetta
13             Misael Varella
15             Ônyx Lorenzoni
30               Afonso Motta
31                  Dagoberto
33      Félix Mendonça Júnior
34              Flávia Morais
36            Giovani Cherini
40             Mário Heringer
42               Roberto Góes
44             Sergio Vidigal
46             Weverton Rocha
48               André Fufuca
51               Diego Garcia
52              Kaio Maniçoba
53                Marcelo Aro
54       Abel Mesquita Júnior
58             Assis do Couto
60     Carlos Henrique Gaguim
62              Domingos Neto
63              Fábio Ramalho
65            Juscelino Filho
68     Marcelo Álvaro Antônio
72           Valtenir Pereira
                ...          
378                     Rocha
410         Gabriel Guimarães
421       

#### A maior bancada é do PMDB(MDB), depois PP, PSDB, etc.

In [34]:
rural.partido.value_counts()

PMDB    45
PP      19
PSDB    18
PSD     17
PR      16
PSB     13
SD      12
DEM      9
PMB      9
PDT      9
PTB      9
PRB      7
PSC      6
PT       6
PHS      3
Pros     2
PPS      2
PV       2
PMN      1
PEN      1
Name: partido, dtype: int64

#### Os estados dos ruralistas, todos com maior concentração em Minas Gerais

In [35]:
rural.estado.value_counts()

MG    30
PR    20
RS    15
SP    14
RJ    12
GO    11
BA    10
PA    10
SC    10
MT     7
TO     6
PB     6
CE     6
MS     6
ES     6
MA     5
AC     5
RO     4
RN     4
AL     3
AP     3
DF     3
PI     3
RR     2
SE     2
PE     2
AM     1
Name: estado, dtype: int64

#### Junção das planilhas ruralistas com a planilha de processos judiciais.

In [36]:
merged = pd.merge(ambiente, rural, on='Nome')

In [37]:
merged.head()

Nome             cargo partido_x estado_x  \
0  Adilton Sachetti  Deputado Federal       PRB       MT   
1  Adilton Sachetti  Deputado Federal       PRB       MT   
2  Adilton Sachetti  Deputado Federal       PRB       MT   
3  Adilton Sachetti  Deputado Federal       PRB       MT   
4  Adilton Sachetti  Deputado Federal       PRB       MT   

                   ocorrencia  \
0  Improbidade Administrativa   
1             Crime Eleitoral   
2   Crime de Responsabilidade   
3   Crime de Responsabilidade   
4   Crime de Responsabilidade   

                                           descricao  \
0  O parlamentar é acusado de realiza repasses ir...   
1         Falsidade Ideológica para fins eleitorais.   
2                          Crime de Responsabilidade   
3  Crime de Responsabilidade enquadrado na Lei de...   
4                          Crime de Responsabilidade   

                           status  \
0                    Em Andamento   
1                    Em Andamento   
2  Em Andamento - Denúncia Aceita   
3                    Em Andamento   
4                    Em Andamento   

                                            processo partido_y estado_y  \
0  Ação Penal 947/2015 - STF (http://portal.stf.j...       PSB       MT   
1  4165/2015 - STF (http://portal.stf.jus.br/proc...       PSB       MT   
2  4210/2015 - STF (http://portal.stf.jus.br/proc...       PSB       MT   
3  4632/2017 - STF (http://portal.stf.jus.br/proc...       PSB       MT   
4  4310/2016 - STF (http://portal.stf.jus.br/proc...       PSB       MT   

         ...          Bala Sindical  Direitos Humanos  Mineração  Empresarial  \
0        ...           NaN      NaN               NaN        NaN          1.0   
1        ...           NaN      NaN               NaN        NaN          1.0   
2        ...           NaN      NaN               NaN        NaN          1.0   
3        ...           NaN      NaN               NaN        NaN          1.0   
4        ...           NaN      NaN               NaN        NaN          1.0   

   Parentes  Saúde  Bola  Empreiteiras/construtoras  Total de bancadas  
0       NaN    NaN   NaN                        NaN                2.0  
1       NaN    NaN   NaN                        NaN                2.0  
2       NaN    NaN   NaN                        NaN                2.0  
3       NaN    NaN   NaN                        NaN                2.0  
4       NaN    NaN   NaN                        NaN                2.0  

[5 rows x 24 columns]

In [38]:
merged.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 385 entries, 0 to 384
Data columns (total 24 columns):
Nome                         385 non-null object
cargo                        385 non-null object
partido_x                    385 non-null object
estado_x                     385 non-null object
ocorrencia                   385 non-null object
descricao                    385 non-null object
status                       384 non-null object
processo                     384 non-null object
partido_y                    385 non-null object
estado_y                     385 non-null object
mandatos                     385 non-null float64
Situação                     331 non-null object
Evangélica                   151 non-null float64
Ruralista                    385 non-null float64
Bala                         20 non-null float64
Sindical                     4 non-null float64
Direitos Humanos             0 non-null float64
Mineração                    12 non-null float64
Empresarial 

#### Noventa deputados da bancada ruralista na Câmara são réus e/ou condenados em 385 processos judiciais.

In [39]:
merged.Ruralista.value_counts()

1.0    385
Name: Ruralista, dtype: int64

In [40]:
merged.groupby(["Nome", "Ruralista"]).ocorrencia.sum().reset_index().sort_values("ocorrencia", ascending=False)

Nome  Ruralista  \
60                Mandetta        1.0   
7             Aníbal Gomes        1.0   
87      Wellington Roberto        1.0   
86        Valtenir Pereira        1.0   
46             Jozi Araújo        1.0   
9              Arthur Lira        1.0   
69           Nelson Meurer        1.0   
74             Paulo Feijó        1.0   
10    Arthur Oliveira Maia        1.0   
1           Aelton Freitas        1.0   
61  Marcelo Álvaro Antônio        1.0   
67            Milton Monti        1.0   
42          Jorginho Mello        1.0   
41             Jony Marcos        1.0   
24           Daniel Vilela        1.0   
52        Leonardo Quintão        1.0   
53         Lincoln Portela        1.0   
48          João Rodrigues        1.0   
21          Célio Silveira        1.0   
65            Marx Beltrão        1.0   
0         Adilton Sachetti        1.0   
83          Simone Morgado        1.0   
19          Celso Maldaner        1.0   
14            Bruna Furlan        1.0   
28           Domingos Neto        1.0   
55      Luis Carlos Heinze        1.0   
39             Hélio Leite        1.0   
70           Nilson Leitão        1.0   
13       Benjamin Maranhão        1.0   
82            Silas Câmara        1.0   
..                     ...        ...   
17           Carlos Melles        1.0   
66             Mauro Lopes        1.0   
89              Átila Lira        1.0   
25            Danilo Forte        1.0   
32         Esperidião Amin        1.0   
45          Jovair Arantes        1.0   
31        Eli Corrêa Filho        1.0   
76             Pedro Uczai        1.0   
51       Leonardo Monteiro        1.0   
68      Nelson Marquezelli        1.0   
15          Carlos Bezerra        1.0   
84           Ságuas Moraes        1.0   
36     Gilberto Nascimento        1.0   
63          Marcus Pestana        1.0   
58          Lázaro Botelho        1.0   
16  Carlos Henrique Gaguim        1.0   
75         Paulo Magalhães        1.0   
54           Luciano Ducci        1.0   
49             Júlio Cesar        1.0   
37         Guilherme Mussi        1.0   
47             João Campos        1.0   
12             Benito Gama        1.0   
4          Alexandre Leite        1.0   
3            Alex Canziani        1.0   
79        Roberto Balestra        1.0   
2            Alceu Moreira        1.0   
35         Geraldo Resende        1.0   
56            Luiz Cláudio        1.0   
80                   Rocha        1.0   
33           Evandro Roman        1.0   

                                           ocorrencia  
60  Tráfico de InfluênciaImprobidade Administrativ...  
7   Sonegação de impostosCrime de Responsabilidade...  
87                            Reivindicação de Imóvel  
86  PeculatoCrime EleitoralCrime EleitoralCrime El...  
46                                     Lesão Corporal  
9   Lavagem de DinheiroLavagem de DinheiroLavagem ...  
69  Lavagem de DinheiroLavagem de DinheiroCorrupçã...  
74      Lavagem de DinheiroDesvio de dinheiro público  
10  Lavagem de DinheiroCrime de ResponsabilidadeIn...  
1   Lavagem de DinheiroCrime de ResponsabilidadeCr...  
61               Investigação PenalInvestigação Penal  
67        Investigação PenalCrime de Responsabilidade  
42                                 Investigação Penal  
41                                 Investigação Penal  
24                                 Investigação Penal  
52                            Infração Administrativa  
53  Improbidade AdministrativaInadimplementoImprob...  
48  Improbidade AdministrativaImprobidade Administ...  
21  Improbidade AdministrativaImprobidade Administ...  
65  Improbidade AdministrativaImprobidade Administ...  
0   Improbidade AdministrativaCrime EleitoralCrime...  
83          Improbidade AdministrativaCrime Eleitoral  
19                         Improbidade Administrativa  
14                         Improbidade Administrativa  
28                         Improbidade Administrativa  
55                     

In [41]:
crime_rural = merged.groupby(["Nome", "Ruralista"]).ocorrencia.sum().reset_index().sort_values("ocorrencia", ascending=False)

In [42]:
crime_rural.head()                    #deputados com processos da bancada ruralista

Nome  Ruralista  \
60            Mandetta        1.0   
7         Aníbal Gomes        1.0   
87  Wellington Roberto        1.0   
86    Valtenir Pereira        1.0   
46         Jozi Araújo        1.0   

                                           ocorrencia  
60  Tráfico de InfluênciaImprobidade Administrativ...  
7   Sonegação de impostosCrime de Responsabilidade...  
87                            Reivindicação de Imóvel  
86  PeculatoCrime EleitoralCrime EleitoralCrime El...  
46                                     Lesão Corporal

In [43]:
crime_rural.Nome.count()

90

In [44]:
crime_rural = merged.groupby(["Nome", "Ruralista"]).ocorrencia.max().reset_index().sort_values("ocorrencia")

In [45]:
crime_rural

Nome  Ruralista  \
33           Evandro Roman        1.0   
80                   Rocha        1.0   
47             João Campos        1.0   
63          Marcus Pestana        1.0   
75         Paulo Magalhães        1.0   
36     Gilberto Nascimento        1.0   
49             Júlio Cesar        1.0   
37         Guilherme Mussi        1.0   
58          Lázaro Botelho        1.0   
12             Benito Gama        1.0   
56            Luiz Cláudio        1.0   
54           Luciano Ducci        1.0   
4          Alexandre Leite        1.0   
3            Alex Canziani        1.0   
84           Ságuas Moraes        1.0   
79        Roberto Balestra        1.0   
34        Genecias Noronha        1.0   
88          Weverton Rocha        1.0   
35         Geraldo Resende        1.0   
72             Osmar Terra        1.0   
31        Eli Corrêa Filho        1.0   
20            Cleber Verde        1.0   
8            Arnaldo Jordy        1.0   
64          Marcus Vicente        1.0   
30            Edmar Arruda        1.0   
23           César Messias        1.0   
78          Ricardo Barros        1.0   
22             César Halum        1.0   
29              Edio Lopes        1.0   
89              Átila Lira        1.0   
..                     ...        ...   
28           Domingos Neto        1.0   
43              Josi Nunes        1.0   
38        Herculano Passos        1.0   
39             Hélio Leite        1.0   
45          Jovair Arantes        1.0   
2            Alceu Moreira        1.0   
53         Lincoln Portela        1.0   
52        Leonardo Quintão        1.0   
51       Leonardo Monteiro        1.0   
61  Marcelo Álvaro Antônio        1.0   
67            Milton Monti        1.0   
41             Jony Marcos        1.0   
24           Daniel Vilela        1.0   
42          Jorginho Mello        1.0   
5           Alfredo Kaefer        1.0   
9              Arthur Lira        1.0   
1           Aelton Freitas        1.0   
10    Arthur Oliveira Maia        1.0   
69           Nelson Meurer        1.0   
74             Paulo Feijó        1.0   
46             Jozi Araújo        1.0   
40            Izalci Lucas        1.0   
85        Toninho Pinheiro        1.0   
86        Valtenir Pereira        1.0   
6           Antônio Jácome        1.0   
70           Nilson Leitão        1.0   
87      Wellington Roberto        1.0   
7             Aníbal Gomes        1.0   
11            Baleia Rossi        1.0   
60                Mandetta        1.0   

                                     ocorrencia  
33                               Abuso de Poder  
80                         Apropriação Indébita  
47                              Crime Eleitoral  
63                              Crime Eleitoral  
75                              Crime Eleitoral  
36                              Crime Eleitoral  
49                              Crime Eleitoral  
37                              Crime Eleitoral  
58                              Crime Eleitoral  
12                              Crime Eleitoral  
56                              Crime Eleitoral  
54                              Crime Eleitoral  
4                               Crime Eleitoral  
3                               Crime Eleitoral  
84                              Crime Eleitoral  
79                              Crime Eleitoral  
34             Crime contra a Lei de Licitações  
88             Crime contra a Lei de Licitações  
35                    Crime de Responsabilidade  
72                    Crime de Responsabilidade  
31                    Crime de Responsabilidade  
20                    Crime de Responsabilidade  
8               Crimes contra a Honra | Calúnia  
64  Crimes contra o Sistema Financeiro Nacional  
30                               Dano Ambiental  
23                               Dano ao Erário  
78                               Dano ao Erário  
22                               Dano ao Erário  
29                    Desvio 

In [46]:
crime_rural[crime_rural.Nome == 'Adilton Sachetti']

Nome  Ruralista                  ocorrencia
0  Adilton Sachetti        1.0  Improbidade Administrativa

In [47]:
crime_rural[crime_rural.Nome == 'Alfredo Kaefer']

Nome  Ruralista           ocorrencia
5  Alfredo Kaefer        1.0  Lavagem de Dinheiro

In [48]:
crime_rural[crime_rural.Nome == 'Dilceu Sperafico']

Empty DataFrame
Columns: [Nome, Ruralista, ocorrencia]
Index: []

In [49]:
crime_rural[crime_rural.Nome == 'Nelson Marquezelli']

Nome  Ruralista                  ocorrencia
68  Nelson Marquezelli        1.0  Improbidade Administrativa

In [50]:
crime_rural[crime_rural.Nome == 'Jorge Amanajás']

Empty DataFrame
Columns: [Nome, Ruralista, ocorrencia]
Index: []

In [51]:
crime_rural[crime_rural.Nome == 'Washigton Reis']

Empty DataFrame
Columns: [Nome, Ruralista, ocorrencia]
Index: []

In [52]:
crime_rural[crime_rural.Nome == 'Gabriel Guimarães']

Empty DataFrame
Columns: [Nome, Ruralista, ocorrencia]
Index: []

In [53]:
crime_rural[crime_rural. Nome == 'Evandro Roman']

Nome  Ruralista      ocorrencia
33  Evandro Roman        1.0  Abuso de Poder

#### Cruzando mais dados, projetos de lei ambientais.
###### Fonte.: https://ruralometro.reporterbrasil.org.br/metodologia#download

In [54]:
contra = pd.read_csv('projetos_ambiente.csv')

In [55]:
contra.head()

Projeto                     Nome          Autor  \
0   PL 334/2015  Marco Tebaldi (PSDB/SC)  MARCO TEBALDI   
1  PL 3200/2015    Covatti Filho (PP/RS)  COVATTI FILHO   
2  PL 1216/2015    Covatti Filho (PP/RS)  COVATTI FILHO   
3  PL 3615/2012       Padre João (PT/MG)     PADRE JOÃO   
4  PL 6442/2016  Nilson Leitão (PSDB/MT)  NILSON LEITÃO   

                                              Ementa  \
0  Altera o art. 4º da Lei nº 1283 de 18 de dezem...   
1  Dispõe sobre a Política Nacional de Defensivos...   
2  Regulamenta o artigo 231 da Constituição Feder...   
3  Acrescenta dispositivo à Lei nº 7.802, de 11 d...   
4  Institui normas reguladoras do trabalho rural ...   

                                             Assunto  Indicado_por  \
0  Permite que estados e municípios realizem insp...        Contag   
1  Cria outro nome para agrotóxicos e facilita a ...        Contag   
2  Cria mais etapas para a demarcação de áreas in...          Apib   
3              Altera regras para pulverização aérea        Contag   
4  Altera regulamentação do trabalho rural e incl...  Contag e ISA   

   Quem_avaliou Qual_avaliacao             Estado          CPF  \
0        Contag      Favorável     SANTA CATARINA  25671235049   
1        Contag   Desfavorável  RIO GRANDE DO SUL   1171688075   
2          Apib   Desfavorável  RIO GRANDE DO SUL   1171688075   
3        Contag      Favorável       MINAS GERAIS  72425610600   
4  Contag e ISA   Desfavorável        MATO GROSSO  34577521172   

  Partido_eleicao Nome_urna_tramitacao  
0            PSDB        MARCO TEBALDI  
1              PP        COVATTI FILHO  
2              PP        COVATTI FILHO  
3              PT           PADRE JOÃO  
4            PSDB        NILSON LEITÃO

In [56]:
contra.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 131 entries, 0 to 130
Data columns (total 12 columns):
Projeto                 131 non-null object
Nome                    131 non-null object
Autor                   131 non-null object
Ementa                  36 non-null object
Assunto                 131 non-null object
Indicado_por            123 non-null object
Quem_avaliou            131 non-null object
Qual_avaliacao          131 non-null object
Estado                  131 non-null object
CPF                     131 non-null int64
Partido_eleicao         131 non-null object
Nome_urna_tramitacao    131 non-null object
dtypes: int64(1), object(11)
memory usage: 12.4+ KB


In [57]:
contra.Assunto.value_counts()

Altera processo de demarcação de terras indígenas                                                                                                               8
Cria plano de sustentabilidade ambiental para municípios de até 100 mil habitantes                                                                              4
Cria CPI para investigar atuação da Funai e do Incra                                                                                                            4
Permite às comunidades indígenas praticar e comercializar atividades agropecuárias e florestais em suas terras                                                  2
Inclui novos procedimentos à Política Nacional sobre Mudança do Clima para redução dos gases de efeito estufa                                                   2
Cria mais etapas para a demarcação de áreas indígenas                                                                                                           2
Suspende regularização do te

In [96]:
contra[contra.Nome == 'Adilton Sachetti']

Empty DataFrame
Columns: [Projeto, Nome, Autor, Ementa, Assunto, Indicado_por, Quem_avaliou, Qual_avaliacao, Estado, CPF, Partido_eleicao, Nome_urna_tramitacao]
Index: []

In [59]:
contra.Qual_avaliacao.value_counts()

Desfavorável    87
Favorável       44
Name: Qual_avaliacao, dtype: int64

In [60]:
contra[contra.Qual_avaliacao == 'Desfavorável'].head()

Projeto                     Nome          Autor  \
1   PL 3200/2015    Covatti Filho (PP/RS)  COVATTI FILHO   
2   PL 1216/2015    Covatti Filho (PP/RS)  COVATTI FILHO   
4   PL 6442/2016  Nilson Leitão (PSDB/MT)  NILSON LEITÃO   
5  PDC 1300/2013  Nilson Leitão (PSDB/MT)  NILSON LEITÃO   
6  PDC 1465/2014  Nilson Leitão (PSDB/MT)  NILSON LEITÃO   

                                              Ementa  \
1  Dispõe sobre a Política Nacional de Defensivos...   
2  Regulamenta o artigo 231 da Constituição Feder...   
4  Institui normas reguladoras do trabalho rural ...   
5  Susta a aplicação da Instrução Normativa nº 01...   
6  Susta a aplicação da Portaria Interministerial...   

                                             Assunto     Indicado_por  \
1  Cria outro nome para agrotóxicos e facilita a ...           Contag   
2  Cria mais etapas para a demarcação de áreas in...             Apib   
4  Altera regulamentação do trabalho rural e incl...     Contag e ISA   
5  Retira da Funai prerrogativa de fazer exigênci...       Cimi e ISA   
6  Institui mudanças no processo de licenciamento...  Cimi, ISA e CPT   

      Quem_avaliou Qual_avaliacao             Estado          CPF  \
1           Contag   Desfavorável  RIO GRANDE DO SUL   1171688075   
2             Apib   Desfavorável  RIO GRANDE DO SUL   1171688075   
4     Contag e ISA   Desfavorável        MATO GROSSO  34577521172   
5       Cimi e ISA   Desfavorável        MATO GROSSO  34577521172   
6  Cimi, ISA e CPT   Desfavorável        MATO GROSSO  34577521172   

  Partido_eleicao Nome_urna_tramitacao  
1              PP        COVATTI FILHO  
2              PP        COVATTI FILHO  
4            PSDB        NILSON LEITÃO  
5            PSDB        NILSON LEITÃO  
6            PSDB        NILSON LEITÃO

In [61]:
proj_contra = contra[contra.Qual_avaliacao == 'Desfavorável']

In [62]:
proj_contra.head(2)

Projeto                   Nome          Autor  \
1  PL 3200/2015  Covatti Filho (PP/RS)  COVATTI FILHO   
2  PL 1216/2015  Covatti Filho (PP/RS)  COVATTI FILHO   

                                              Ementa  \
1  Dispõe sobre a Política Nacional de Defensivos...   
2  Regulamenta o artigo 231 da Constituição Feder...   

                                             Assunto Indicado_por  \
1  Cria outro nome para agrotóxicos e facilita a ...       Contag   
2  Cria mais etapas para a demarcação de áreas in...         Apib   

  Quem_avaliou Qual_avaliacao             Estado         CPF Partido_eleicao  \
1       Contag   Desfavorável  RIO GRANDE DO SUL  1171688075              PP   
2         Apib   Desfavorável  RIO GRANDE DO SUL  1171688075              PP   

  Nome_urna_tramitacao  
1        COVATTI FILHO  
2        COVATTI FILHO

In [63]:
proj_contra.Autor.value_counts()

JERÔNIMO GOERGEN            9
LUIS CARLOS HEINZE          8
NILSON LEITÃO               8
VALDIR COLATTO              7
JOSUÉ BENGTSON              5
ADILTON SACHETTI            3
ALCEU MOREIRA               3
IRAJÁ ABREU                 3
ROBERTO BALESTRA            2
MARCOS MONTES               2
EDUARDO DA FONTE            2
DOMINGOS SÁVIO              2
CARLOS BEZERRA              2
AFONSO HAMM                 2
VICENTINHO JÚNIOR           2
COVATTI FILHO               2
FLAVIANO MELO               1
DILCEU SPERAFICO            1
WELITON PRADO               1
JOVAIR ARANTES              1
CARLOS HENRIQUE GAGUIM      1
BETO FARO                   1
ALFREDO KAEFER              1
MAX FILHO                   1
SERGIO SOUZA                1
LAERCIO OLIVEIRA            1
CELSO MALDANER              1
PROFESSOR VICTÓRIO GALLI    1
DR. JORGE SILVA             1
ANTONIO BALHMANN            1
HEITOR SCHUCH               1
LUCIO MOSQUINI              1
JOÃO CARLOS BACELAR         1
TEREZA CRI

In [64]:
proj_contra.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 87 entries, 1 to 130
Data columns (total 12 columns):
Projeto                 87 non-null object
Nome                    87 non-null object
Autor                   87 non-null object
Ementa                  34 non-null object
Assunto                 87 non-null object
Indicado_por            80 non-null object
Quem_avaliou            87 non-null object
Qual_avaliacao          87 non-null object
Estado                  87 non-null object
CPF                     87 non-null int64
Partido_eleicao         87 non-null object
Nome_urna_tramitacao    87 non-null object
dtypes: int64(1), object(11)
memory usage: 8.8+ KB


In [65]:
proj_contra.groupby('Autor')['Assunto'].value_counts()

Autor                     Assunto                                                                                                                                                     
ADILTON SACHETTI          Autoriza aproveitamento da água em trechos das hidrovias dos rios Tapajós, Teles Pires e Juruena                                                                1
                          Autoriza aproveitamento da água na hidrovia do rio Paraguai                                                                                                     1
                          Autoriza obras para construção de hidrovias nos rios Tocantins (PA), Araguaia (GO) e das Mortes (MT), inclusive em terras indígenas                             1
AFONSO HAMM               Determina que recomposição de áreas de preservação seja feita com plantio de árvores frutíferas; hoje é feita espécies nativas                                  1
                          Libera o porte de arma para trabalhador

In [66]:
pl = pd.merge(rural, proj_contra, on="Nome")

In [67]:
pl.head(3)

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
0  Giovani Cherini     PDT     RS       2.0       NaN         NaN        1.0   
1    Alceu Moreira    PMDB     RS       2.0  Reeleito         1.0        1.0   
2   Carlos Bezerra    PMDB     MT       4.0  Reeleito         NaN        1.0   

   Bala  Sindical  Direitos Humanos         ...                     Autor  \
0   NaN       NaN               NaN         ...           GIOVANI CHERINI   
1   1.0       NaN               NaN         ...             ALCEU MOREIRA   
2   NaN       NaN               NaN         ...            CARLOS BEZERRA   

   Ementa                                            Assunto  Indicado_por  \
0     NaN  Anula efeitos de portaria do Ministério do Tra...           ISA   
1     NaN  Anula portaria do Ministério do Meio Ambiente ...           ISA   
2     NaN  Proposta dificulta desapropriação de terras pa...     ISA e CPT   

   Quem_avaliou  Qual_avaliacao             Estado          CPF  \
0           ISA    Desfavorável  RIO GRANDE DO SUL  31050778049   
1           ISA    Desfavorável  RIO GRANDE DO SUL  17968496004   
2     ISA e CPT    Desfavorável        MATO GROSSO    834939134   

  Partido_eleicao Nome_urna_tramitacao  
0             PDT      GIOVANI CHERINI  
1            PMDB        ALCEU MOREIRA  
2            PMDB       CARLOS BEZERRA  

[3 rows x 28 columns]

In [68]:
pl.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 22 entries, 0 to 21
Data columns (total 28 columns):
Nome                         22 non-null object
partido                      22 non-null object
estado                       22 non-null object
mandatos                     22 non-null float64
Situação                     20 non-null object
Evangélica                   11 non-null float64
Ruralista                    22 non-null float64
Bala                         1 non-null float64
Sindical                     1 non-null float64
Direitos Humanos             0 non-null float64
Mineração                    0 non-null float64
Empresarial                  16 non-null float64
Parentes                     14 non-null float64
Saúde                        0 non-null float64
Bola                         1 non-null float64
Empreiteiras/construtoras    7 non-null float64
Total de bancadas            22 non-null float64
Projeto                      22 non-null object
Autor                      

In [69]:
pl.Quem_avaliou.value_counts()

ISA          20
ISA e CPT     2
Name: Quem_avaliou, dtype: int64

In [70]:
pl[pl.Assunto == 'Determina que recomposição de áreas de preservação seja feita com plantio de árvores frutíferas; hoje é feita espécies nativas']

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
8  Afonso Hamm      PP     RS       4.0  Reeleito         NaN        1.0   

   Bala  Sindical  Direitos Humanos         ...                 Autor  Ementa  \
8   NaN       NaN               NaN         ...           AFONSO HAMM     NaN   

                                             Assunto  Indicado_por  \
8  Determina que recomposição de áreas de preserv...           ISA   

   Quem_avaliou  Qual_avaliacao             Estado          CPF  \
8           ISA    Desfavorável  RIO GRANDE DO SUL  37040642034   

  Partido_eleicao Nome_urna_tramitacao  
8              PP          AFONSO HAMM  

[1 rows x 28 columns]

In [71]:
pl[pl.Assunto == 'Anula resolução do Conama que prevê a proteção de área de preservação permanente no entorno de reservatórios']

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
19  Josué Bengtson     PTB     PA       4.0  Reeleito         1.0        1.0   

    Bala  Sindical  Direitos Humanos         ...                    Autor  \
19   NaN       NaN               NaN         ...           JOSUÉ BENGTSON   

    Ementa                                            Assunto  Indicado_por  \
19     NaN  Anula resolução do Conama que prevê a proteção...           ISA   

    Quem_avaliou  Qual_avaliacao  Estado         CPF Partido_eleicao  \
19           ISA    Desfavorável    PARÁ  9673504768             PTB   

   Nome_urna_tramitacao  
19       JOSUÉ BENGTSON  

[1 rows x 28 columns]

In [72]:
pl[pl.Assunto == 'Proposta dificulta desapropriação de terras para reforma agrária']

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
2  Carlos Bezerra    PMDB     MT       4.0  Reeleito         NaN        1.0   

   Bala  Sindical  Direitos Humanos         ...                    Autor  \
2   NaN       NaN               NaN         ...           CARLOS BEZERRA   

   Ementa                                            Assunto  Indicado_por  \
2     NaN  Proposta dificulta desapropriação de terras pa...     ISA e CPT   

   Quem_avaliou  Qual_avaliacao       Estado        CPF Partido_eleicao  \
2     ISA e CPT    Desfavorável  MATO GROSSO  834939134            PMDB   

  Nome_urna_tramitacao  
2       CARLOS BEZERRA  

[1 rows x 28 columns]

In [73]:
pl[pl.Assunto == 'Dispensa licença ambiental para agropecuária em áreas degradadas']

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
14  Irajá Abreu     PSD     TO       2.0  Reeleito         NaN        1.0   

    Bala  Sindical  Direitos Humanos         ...                 Autor  \
14   NaN       NaN               NaN         ...           IRAJÁ ABREU   

    Ementa                                            Assunto  Indicado_por  \
14     NaN  Dispensa licença ambiental para agropecuária e...           ISA   

    Quem_avaliou  Qual_avaliacao     Estado          CPF Partido_eleicao  \
14           ISA    Desfavorável  TOCANTINS  60507306104             PSD   

   Nome_urna_tramitacao  
14          IRAJÁ ABREU  

[1 rows x 28 columns]

In [74]:
pl[pl.Assunto == 'Susta portaria que proíbe coleta, corte e comercialização de espécies da flora ameaçadas de extinção']

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
16  Nilson Leitão    PSDB     MT       2.0  Reeleito         NaN        1.0   

    Bala  Sindical  Direitos Humanos         ...                   Autor  \
16   NaN       NaN               NaN         ...           NILSON LEITÃO   

    Ementa                                            Assunto  Indicado_por  \
16     NaN  Susta portaria que proíbe coleta, corte e come...           ISA   

    Quem_avaliou  Qual_avaliacao       Estado          CPF Partido_eleicao  \
16           ISA    Desfavorável  MATO GROSSO  34577521172            PSDB   

   Nome_urna_tramitacao  
16        NILSON LEITÃO  

[1 rows x 28 columns]

In [75]:
pl[pl.Assunto == 'Anula portaria do Ministério do Meio Ambiente que proíbe pesca de 475 espécies ameaçadas de extinção']

Nome partido estado  mandatos  Situação  Evangélica  Ruralista  \
1  Alceu Moreira    PMDB     RS       2.0  Reeleito         1.0        1.0   

   Bala  Sindical  Direitos Humanos         ...                   Autor  \
1   1.0       NaN               NaN         ...           ALCEU MOREIRA   

   Ementa                                            Assunto  Indicado_por  \
1     NaN  Anula portaria do Ministério do Meio Ambiente ...           ISA   

   Quem_avaliou  Qual_avaliacao             Estado          CPF  \
1           ISA    Desfavorável  RIO GRANDE DO SUL  17968496004   

  Partido_eleicao Nome_urna_tramitacao  
1            PMDB        ALCEU MOREIRA  

[1 rows x 28 columns]

In [76]:
pl.Autor.value_counts()

JOSUÉ BENGTSON        3
VALDIR COLATTO        2
CARLOS BEZERRA        2
IRAJÁ ABREU           1
NILSON LEITÃO         1
TONINHO PINHEIRO      1
TEREZA CRISTINA       1
AFONSO HAMM           1
SERGIO SOUZA          1
JOVAIR ARANTES        1
ALFREDO KAEFER        1
HEITOR SCHUCH         1
LAERCIO OLIVEIRA      1
CELSO MALDANER        1
ALCEU MOREIRA         1
ROBERTO BALESTRA      1
GIOVANI CHERINI       1
LUIS CARLOS HEINZE    1
Name: Autor, dtype: int64

In [77]:
pl.groupby('Autor')['Assunto'].value_counts()

Autor               Assunto                                                                                                                                                 
AFONSO HAMM         Determina que recomposição de áreas de preservação seja feita com plantio de árvores frutíferas; hoje é feita espécies nativas                              1
ALCEU MOREIRA       Anula portaria do Ministério do Meio Ambiente que proíbe pesca de 475 espécies ameaçadas de extinção                                                        1
ALFREDO KAEFER      Determina que ANP regule e fiscalize exploração de xisto betuminoso                                                                                         1
CARLOS BEZERRA      Classifica cavernas como áreas de proteção e patrimônio nacional                                                                                            1
                    Proposta dificulta desapropriação de terras para reforma agrária                               

#### Cruzando com dados de deputados multados pelo Ibama; deputados com empresas rurais e doações de empresas flagradas por trabalho escravo.
#### Fonte.: https://ruralometro.reporterbrasil.org.br/metodologia#download

In [78]:
ibama = pd.read_csv('multados_ibama.csv')

In [79]:
ibama

deputado_nome Entidade_documento_original  \
0   Carlos Cesar Correia de Messias              508.720.607-72   
1             Irajá Silvestre Filho              605.073.061-04   
2             Irajá Silvestre Filho              605.073.061-04   
3             Irajá Silvestre Filho              605.073.061-04   
4                  Jose Alves Rocha              047.891.135-15   
5          José Maria Macedo Júnior              197.057.864-53   
6          José Maria Macedo Júnior              197.057.864-53   
7          José Maria Macedo Júnior              197.057.864-53   
8          José Maria Macedo Júnior              197.057.864-53   
9          José Maria Macedo Júnior              197.057.864-53   
10         José Maria Macedo Júnior              197.057.864-53   
11         José Maria Macedo Júnior              197.057.864-53   
12            Luiz Gonzaga Patriota              019.609.704-53   

    Entidade_documento Infracao_Ibama_Estado_Id Infracao_Ibama_Municipio  \
0          50872060772                       AC           Sena Madureira   
1          60507306104                       TO     Aliança do Tocantins   
2          60507306104                       TO     Aliança do Tocantins   
3          60507306104                       TO     Aliança do Tocantins   
4           4789113515                       BA                   Coribe   
5          19705786453                       CE                Fortaleza   
6          19705786453                       CE                Fortaleza   
7          19705786453                       CE                Fortaleza   
8          19705786453                       CE                Fortaleza   
9          19705786453                       CE                  Iracema   
10         19705786453                       CE                  Iracema   
11         19705786453                       CE                   Fortim   
12          1960970453                       PE                Petrolina   

                         Infracao_Ibama_Tipo_Infracao  \
0                                   Infração da flora   
1   Destruir ou danificar florestas, cortar árvore...   
2   Destruir, desmatar, danificar florestas ou qua...   
3   Descumprir o Termo de Embargo de n° 568531-C, ...   
4   Desmatar florestas ou demais formas de vegetaç...   
5   Construir um galpão e um duplex em um terreno ...   
6   Instalar uma construção de um galpão e um apar...   
7   Utilizar 2,14 ha de área considerada de preser...   
8   Deixar de atender a exigências legais quando d...   
9   Destruir 2,2 ha de vegetação natural e abertur...   
10  Construir obra de 400 metros quadrado localiza...   
11  Descumprir embargo de uma construção de um gal...   
12  Desmatar 10,8360 Hectares de floresta nativa (...   

   Infracao_Ibama_Data_Lista  Valor_multa  
0                 20/10/2015          6.0  
1                  19/5/2010         65.0  
2                  19/5/2010         55.0  
3                  19/5/2010         10.0  
4                  14/3/2008          5.0  
5                        NaN         30.0  
6                        NaN         20.0  
7                        NaN         15.0  
8                        NaN         15.0  
9                  17/5/2010         30.0  
10                 17/5/2010        150.0  
11                       NaN         20.0  
12                  4/4/2017         11.0

In [80]:
ibama.deputado_nome.value_counts()

José Maria Macedo Júnior           7
Irajá Silvestre Filho              3
Luiz Gonzaga Patriota              1
Jose Alves Rocha                   1
Carlos Cesar Correia de Messias    1
Name: deputado_nome, dtype: int64

In [81]:
ibama.Valor_multa.max()

150.0

In [82]:
ibama.groupby('deputado_nome')['Valor_multa'].sum()

deputado_nome
Carlos Cesar Correia de Messias      6.0
Irajá Silvestre Filho              130.0
Jose Alves Rocha                     5.0
José Maria Macedo Júnior           280.0
Luiz Gonzaga Patriota               11.0
Name: Valor_multa, dtype: float64

In [83]:
escravo = pd.read_csv('dep_emp_escravo.csv')

In [84]:
escravo.politico_nome_urna

0          BETO MANSUR
1          BETO MANSUR
2    LEONARDO PICCIANI
Name: politico_nome_urna, dtype: object

In [85]:
dep_rural = pd.read_csv('empresas_rurais_dep.csv')

In [86]:
dep_rural.head()

uf                        Nome           cpf nome_urna partido_eleicao  \
0  PA  JOSE BENITO PRIANTE JUNIOR  1.729362e+10   PRIANTE            PMDB   
1  PA  JOSE BENITO PRIANTE JUNIOR  1.729362e+10   PRIANTE            PMDB   
2  PA  JOSE BENITO PRIANTE JUNIOR  1.729362e+10   PRIANTE            PMDB   
3  PA  JOSE BENITO PRIANTE JUNIOR  1.729362e+10   PRIANTE            PMDB   
4  PA  JOSE BENITO PRIANTE JUNIOR  1.729362e+10   PRIANTE            PMDB   

             cnpj                                 empresa estado  \
0   4322499000104    AGROPECUARIA RIO DA PRATA LTDA - EPP   Pará   
1   4322499000104    AGROPECUARIA RIO DA PRATA LTDA - EPP   Pará   
2   4322499000104    AGROPECUARIA RIO DA PRATA LTDA - EPP   Pará   
3  20008829000148  AGROINDUSTRIAL ACAI DA PRATA LTDA - ME   Pará   
4  20008829000148  AGROINDUSTRIAL ACAI DA PRATA LTDA - ME   Pará   

                          socios situacao_empresa                cidade  \
0     JOSE BENITO PRIANTE JUNIOR            ativa  Santa Isabel do Pará   
1       JOSE BENITO PRIANTE NETO            ativa  Santa Isabel do Pará   
2  LILIANE PEREIRA PINTO PRIANTE            ativa  Santa Isabel do Pará   
3     JOSE BENITO PRIANTE JUNIOR            ativa  Santa Isabel do Pará   
4       JOSE BENITO PRIANTE NETO            ativa  Santa Isabel do Pará   

                 cnae_principal propriedade capital_social  \
0  01.33-4-01 - Cultivo de açaí         sim         750000   
1  01.33-4-01 - Cultivo de açaí         sim         750000   
2  01.33-4-01 - Cultivo de açaí         sim         750000   
3  01.33-4-01 - Cultivo de açaí         sim          50000   
4  01.33-4-01 - Cultivo de açaí         sim          50000   

  agricola_ou_relacionada  cnpj_raiz  
0                     sim    4322499  
1                     sim    4322499  
2                     sim    4322499  
3                     sim   20008829  
4                     sim   20008829

In [87]:
dep_rural.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 167 entries, 0 to 166
Data columns (total 16 columns):
uf                         165 non-null object
Nome                       167 non-null object
cpf                        165 non-null float64
nome_urna                  167 non-null object
partido_eleicao            165 non-null object
cnpj                       167 non-null int64
empresa                    167 non-null object
estado                     167 non-null object
socios                     167 non-null object
situacao_empresa           167 non-null object
cidade                     167 non-null object
cnae_principal             167 non-null object
propriedade                167 non-null object
capital_social             167 non-null object
agricola_ou_relacionada    167 non-null object
cnpj_raiz                  167 non-null int64
dtypes: float64(1), int64(2), object(13)
memory usage: 21.0+ KB


In [88]:
dep_rural.Nome.value_counts()

ANIBAL FERREIRA GOMES                    10
LUIS CARLOS HEINZE                        9
NEWTON CARDOSO JUNIOR                     8
PAULO SALIM MALUF                         7
IRAJA SILVESTRE FILHO                     7
CARLOS DO CARMO ANDRADE MELLES            6
CLAUDIO CAJADO SAMPAIO                    6
MARCELO COSTA E CASTRO                    5
JOSE BENITO PRIANTE JUNIOR                5
JOSE ORCIRIO MIRANDA DOS SANTOS           5
GIUSEPPE VECCI                            5
JACOB ALFREDO STOFFELS KAEFER             5
LEONARDO CARNEIRO MONTEIRO PICCIANI       5
LUIZ HILOSHI NISHIMORI                    5
MARIO LUCIO HERINGER                      5
VALDIR LUIZ ROSSONI                       5
EDMAR DE SOUZA ARRUDA                     4
NELSON MARQUEZELLI                        4
ARTHUR DE OLIVEIRA MAIA DA SILVA          4
PAULO ROBERTO GOMES MANSUR                4
MARINALDO ROSENDO DE ALBUQUERQUE          4
JOAO CARLOS PAOLILO BACELAR FILHO         4
FELIX DE ALMEIDA MENDONCA JUNIOR

In [89]:
sujas = pd.read_csv('doacoes_escravo.csv')

In [90]:
sujas.head()

uf             cargo                               Nome          cpf  \
0  MG  Deputado Federal      DOMINGOS SAVIO CAMPOS RESENDE  22275150668   
1  BA  Deputado Federal                LUIZ CARLOS CAETANO   7409419504   
2  BA  Deputado Federal       MOEMA ISABEL PASSOS GRAMACHO  13339982520   
3  BA  Deputado Federal  NELSON VICENTE PORTELA PELLEGRINO  24289612504   
4  BA  Deputado Federal          JOSÉ CARLOS ALELUIA COSTA   1782037500   

   cpf_cnpj_doador                          nome_doador  valor  \
0   23998438000106  CCM - CONSTRUTORA CENTRO MINAS LTDA  15000   
1   16365025000129         GRAFICO EMPREENDIMENTOS LTDA   6000   
2   16365025000129                   GRAFICO ENGENHARIA  10000   
3   16365025000129         GRAFICO EMPREENDIMENTOS LTDA   3000   
4   16365025000129        GRAFICA EMPREENDIMENTOS LTDA.   1000   

  cpf_cnpj_doador_originario nome_doador_originario tipo_doacao  
0                      #NULO                  #NULO      direta  
1                      #NULO                  #NULO      direta  
2                      #NULO                  #NULO      direta  
3                      #NULO                  #NULO      direta  
4                      #NULO                  #NULO      direta

In [91]:
sujas.groupby('Nome')['nome_doador'].value_counts()

Nome                                   nome_doador                                                  
AFONSO BANDEIRA FLORENCE               SERTENGE S/A                                                     1
AGUINALDO VELLOSO BORGES RIBEIRO       Direção Estadual/Distrital                                       2
ALFREDO PEREIRA DO NASCIMENTO          Direção Estadual/Distrital                                       1
ANDRÉ LUIS CARVALHO RIBEIRO            FRANCISCO DANTAS RIBEIRO FILHO                                   1
ANTONIO SERGIO ALVES VIDIGAL           MRV ENGENHARIA E PARTICPAÇÕES SA                                 1
ARLINDO CHIGNALIA JUNIOR               Direção Nacional                                                 1
ARTHUR DE OLIVEIRA MAIA DA SILVA       SINOBRAS - SIDERURGICA NORTE BRASIL - S.A.                       1
ATILA FREITAS LIRA                     CELSO DE SOUSA MENDES FILHO                                      1
CAIO NARCIO RODRIGUES DA SILVEIRA      SUCOCÍTRICO 

In [92]:
sujas.Nome.max()

'WELINGTON COIMBRA'

In [93]:
sujas.Nome.min()

'AFONSO BANDEIRA FLORENCE'

In [94]:
sujas.nome_doador.max()

'TRATENGE ENGENHARIA LTDA'

In [95]:
sujas.nome_doador.value_counts()

ELEICAO 2014 PAULO ANTONIO SKAF GOVERNADOR                       8
PAULO ANTONIO SKAF                                               8
ELEIÇÃO 2014 - PAULO ANTONIO SKAF GOVERNADOR                     8
ELEIÇÃO 2014 PAULO JOSE CARLOS GUEDES                            6
ELEICAO 2014 DURVAL ANGELO ANDRADE DEPUTADO ESTADUAL             6
Direção Estadual/Distrital                                       6
CONSTRUTORA JUREMA LTDA                                          4
Direção Nacional                                                 4
SUCOCITRICO CUTRALE LTDA                                         3
SIDERURGICA NORTE BRASIL S.A                                     3
GRAFICO EMPREENDIMENTOS LTDA                                     3
Comitê Financeiro Único                                          3
ELEIÇÃO 2014 PAULO ROBERTO LAMAC JUNIOR                          3
SERTENGE S/A                                                     3
 ELEIÇÃO 2014 MICHEL MIGUEL ELIAS TEMER LULIA VICE-PRESIDENTE 